core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -48,25 +49,12 @@
   }
 
   private static void formatFolder(String name, String folder) throws IOException {
-    UnderFileSystem ufs = UnderFileSystem.Factory.get(folder);
```

Comment:
this is pure local fs operation, not necessary to use `UnderFileSystem.Factory.get`